test/cpp/end2end/xds/xds_fault_injection_end2end_test.cc

```diff
@@ -204,11 +204,11 @@ TEST_P(FaultInjectionTest, XdsFaultInjectionPercentageAbortViaHeaders) {
 
 TEST_P(FaultInjectionTest, XdsFaultInjectionPercentageDelay) {
   CreateAndStartBackends(1);
-  const uint32_t kRpcTimeoutMilliseconds = grpc_test_slowdown_factor() * 3000;
+  const uint32_t kRpcTimeoutMilliseconds = 10000;
```

Comment:
Don't we still want to use `grpc_test_slowdown_factor()` here, with whatever timeout value we choose?

Reply:
This is used in RpcOptions, and RpcOptions has it's own grpc_test_slowdown_factor() multiplier -
https://github.com/grpc/grpc/blob/2a7286a67c71e2e0c0eee5e49ba1a719fcabb3a8/test/cpp/end2end/xds/xds_end2end_test_lib.cc#L419
https://github.com/grpc/grpc/blob/2a7286a67c71e2e0c0eee5e49ba1a719fcabb3a8/test/core/util/test_config.cc#L81

Reply:
Ah, good catch!